pom.xml

```diff
@@ -6766,8 +6773,10 @@
              <artifactId>maven-surefire-plugin</artifactId>
              <configuration>
                <systemPropertyVariables>
-                 <kafka>true</kafka>
-                 <kafka.bootstrap.servers>${kafka.bootstrap.servers}</kafka.bootstrap.servers>
+                 <kafka>true</kafka> <!-- deprecated since 10.3 -->
```

Comment:
This is referenced in README and used by integration scripts and jobs nuxeo-master-fullbuild-part1-kafka-1.0 and FT-nuxeo-master-server-tomcat-kafka-1.0

Reply:
Agree: https://jira.nuxeo.com/browse/NXP-25770